tests/end2end/cypress/integration/dataviz-ghaction.js

```diff
@@ -74,27 +69,6 @@ describe('Dataviz tests', function () {
             .children().first() // <div class="svg-container" style="position:relative; width: 100%; height: 400px;">
             .should('have.class', 'svg-container')
             .should('have.css', 'height', '400px')
-
-        cy.get('[data-title="Download plot as a png"]').click()
-
-        let expected_diff = 0
-        let fixture_path = 'images/plotly/plot_montpellier_bakeries.png'
-        if (Cypress.isBrowser('firefox')) {
-            fixture_path = 'images/plotly/plot_montpellier_bakeries_firefox.png'
-            expected_diff = 235
-        }
-        cy.fixture(fixture_path).then((expected) => {
-            // newplot.png is maybe not the last one if the download folder is not empty ...
-            cy.readFile(path.join(downloadsFolder, "newplot.png"), 'base64').then((image) => {
-
-                // We can make a easier function to compare two images
-                const img1 = PNG.sync.read(Buffer.from(image, 'base64'));
-                const { width, height } = img1;
-                const img2 = PNG.sync.read(Buffer.from(expected, 'base64'));
-
-                expect(pixelmatch(img1.data, img2.data, null, width, height, { threshold: 0.1 }), 'expect plot').to.lessThan(expected_diff + 1)
```

Comment:
Why removing this test?
For the rest there are so many changes that I think it needs a test to be sure we have the right behavior.

Reply:
This feature is brought by Plotly, and adds unnecessary complexity (behaviour different from FF and other browser, etc). It failed locally, and this kind of pixel comparison is prone to failures. I discussed with @Gustry and @rldhont, authors of the addition of this test, and they agree to remove it.

Reply:
it corresponds to the PNG export of a plot, which is not a Lizmap feature but a Plotly feature. I added some PR ago tests to get the JSON configuration sent by the backend and check the returned data is alright.

Reply:
Testing the JSON from plotly https://github.com/3liz/lizmap-web-client/blob/master/tests/end2end/cypress/integration/dataviz-ghaction.js